llvm/include/llvm/Transforms/Instrumentation/RealtimeSanitizer.h

```diff
@@ -0,0 +1,34 @@
+//=- RealtimeSanitizer.h - RealtimeSanitizer instrumentation ----*- C++ -*-==//
+//-*-===//
```

Comment:
unneeded  second line

the first line should use `//===-` (3 `=`). Both `//===-` and `//===-- ` are common in the code base and I am happy with either style.

Reply:
Apologies, thanks for the double check! clang-format scrambled things a bit, should be fixed when I push this next version up.